packages/js-utils/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@automattic/js-utils",
+	"version": "1.0.0-alpha.0",
+	"description": "General purpose Javascript utility functions",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic Inc.",
+	"sideEffects": true,
```

Comment:
I think ideally this package shouldn't have any side effects. Are we planning any?

Reply:
> Are we planning any?

Nope 👍🏻  fixed in https://github.com/Automattic/wp-calypso/pull/50263/commits/7a468fb549d9ee37b63c278a591880f9755480f4